mqttclients/dotnet/MQTTnet.Client.Extensions/RequestTopicAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace MQTTnet.Client.Extensions
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    public class RequestTopicAttribute : Attribute
+    {
+        public string Topic { get; set; }
+        public RequestTopicAttribute(string topic)
```

Comment:
What's the point of having this method here if `Topic` already can be set on its own?

Reply:
required by the ctor of the attribute